sdks/python/apache_beam/coders/standard_coders.yaml

```diff
@@ -0,0 +1,39 @@
+coder:
+  urn: "beam:coders:bytes:0.1"
```

Comment:
Cover the 5 kind types:
windowed_value
global_window
pair
length_prefix
stream

Based on our current kind types, it seems like we should get a single simple value type which behaves differently when nested vs unnested, "bytes" seems like the right choice.